relentless/variable.py

```diff
@@ -74,13 +86,79 @@ class Variable(abc.ABC):
     not inherit from this variable directly but should make use of the more
     flexible :class:`IndependentVariable` or :class:`DependentVariable` types.
 
+    Examples
+    --------
+    Create variables with specified value::
+
+        x = relentless.variable.IndependentVariable(value=2.0)
+
+    If the value is numerical, arithmetic operations are available::
+
+        >>> print(x+2.0)
```

Comment:
Should we add `__str__` to the `Variable`? I think these examples don't work as shown, since we haven't written a string converter. Maybe you can test and confirm? It would be something small like:
```
def __str__(self):
    return str(self.value)
```